include/fmt/chrono.h

```diff
@@ -1822,6 +1834,18 @@ template <typename Char> struct formatter<std::tm, Char> {
     return end;
   }
 
+  template <typename It>
+  It do_format(It out, const std::tm& tm, const std::locale& loc) const {
```

Comment:
Why introduce this function?

Reply:
To avoid repetion in `foram()` method.

Reply:
Keeping ternary should avoid the repetition and introducing a new helper funtion.